src/error-handling/exercise.rs

```diff
@@ -88,25 +88,30 @@ fn eval(e: Expression) -> Result<i64, DivideByZeroError> {
 // ANCHOR_END: solution
 
 // ANCHOR: tests
-#[test]
-fn test_error() {
-    assert_eq!(
-        eval(Expression::Op {
-            op: Operation::Div,
-            left: Box::new(Expression::Value(99)),
-            right: Box::new(Expression::Value(0)),
-        }),
-        Err(DivideByZeroError)
-    );
-}
+#[cfg(test)]
+mod test {
```

Comment:
I don't think we need to setup the full `mod test` for the exercise tests. It's a best practice in actual Rust projects, but I think it just adds visual noise that is distracting for students. I think it's cleaner to just see the list of tests in the same module. Also we don't bother setting up a `mod test` for most of the other exercises, so this is a bit inconsistent with our other tests.

Reply:
I didn't in most exercises, but this is the segment after the "testing" segment (and the second-to-last segment, the last being "Unsafe Rust" which isn't amenable to tests), so it seems a good place to continue to follow the best practice. So, I'd like to leave this as-is.